packages/mobile/test/schemas.ts

```diff
@@ -866,6 +866,7 @@ export const v26Schema = {
   app: {
     ...v25Schema.app,
     rampCashInButtonExpEnabled: false,
+    dappsExplorerEnabled: false,
```

Comment:
I think that when we add new stuff we need to create a new schema, ie`v27Schema` with the new feature flag.

I think it will also require making a redux store migration.